Makefile

```diff
@@ -1,8 +1,25 @@
-BIN := drone-google-cloudstorage
-IMG ?= plugins/$(BIN)
+.PHONY: clean deps test build docker
 
-docker: $(BIN) Dockerfile
-	docker build --rm -t $(IMG) .
+export GOOS ?= linux
+export GOARCH ?= amd64
+export CGO_ENABLED ?= 0
 
-$(BIN): $(wildcard *.go)
-	GOOS=linux GOARCH=amd64 go build
+CI_BUILD_NUMBER ?= 0
+
+LDFLAGS += -X "main.buildDate=$(shell date -u '+%Y-%m-%d %H:%M:%S %Z')"
+LDFLAGS += -X "main.build=$(CI_BUILD_NUMBER)"
+
+clean:
+	go clean -i ./...
+
+deps:
+	go get -t ./...
+
+test:
+	go test -cover ./...
+
+build:
```

Comment:
why change this to `build`? make won't be able to identify what it's actually building anymore.

another issue with this is there are no target prerequisites so `make build` will always execute unconditionally.